src/index.js

```diff
@@ -1,20 +1,35 @@
 require("dotenv").config();
-const { ApolloServer } = require("apollo-server");
+const { ApolloServer } = require("apollo-server-cloud-functions");
 const typeDefs = require("./schema");
-const resolvers = require("./resolvers");
+const { getResolvers } = require("./resolvers");
+const {
+  getLatestList,
+  getDocument,
+  getOrderedCatalog,
+  addItemToCatalog,
+} = require("./service");
 const mocks = require("./mocks");
 
+const resolvers = await getResolvers(
+  getOrderedCatalog,
+  getLatestList,
+  addItemToCatalog,
+  getDocument
+);
+
 const server = new ApolloServer({
-  typeDefs,
-  // mocks,
-  // mockEntireSchema: false,
-  resolvers,
+  typeDefs: typeDefs,
+  resolvers: resolvers,
+  csrfPrevention: true,
+  cache: "bounded",
```

Comment:
In JavaScript, we can just use the key if the value is the same:

```suggestion
  typeDefs,
  resolvers,
  csrfPrevention: true,
  cache: "bounded",
```